cdap-unit-test/pom.xml

```diff
@@ -113,6 +113,11 @@
       <artifactId>spark-sql_2.10</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.sun.jersey</groupId>
+      <artifactId>jersey-servlet</artifactId>
+      <version>1.18</version>
+    </dependency>
```

Comment:
I think that means for tests of this module - `cdap-unit-test`.
People should already have `cdap-unit-test` as a test scope dependency.
I tried making the jersey-servlet a test scope dependency, but the test case running a spark program (in another project that depended on this) then failed.